doc/chapters/lxc/index.rst

```diff
@@ -76,5 +83,3 @@ LXC API
 -------
 TBD
 
-.. [#lxc] https://linuxcontainers.org/
-.. [#fhs] https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard
```

Comment:
They are in-line instead now. Do we also want to have them in a "Links"/"References" chapter?